db/models.py

```diff
@@ -50,3 +53,62 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to="User",
+        related_name="orders",
```

Comment:
the lecture says it's better to access the settings here. (settings.AUTH_USER_MODEL) not to="User"